info.yml

```diff
@@ -27,5 +27,5 @@ requires_core_version: "v0.14.58"
  
 # the frameworks required to run this app
 frameworks:
-    - {"name": "tk-framework-shotgunutils", "version": "v4.x.x"}
+    - {"name": "tk-framework-shotgunutils", "version": "v5.x.x"}
```

Comment:
I simply bumped the version number, I have to admit I didn't ensure everything was still working correctly. Francis should probably to a pass on the demo app to test all the widgets. But in theory, yes, this is the last package that depended on v4 as far as I can tell.